src/main/java/me/totalfreedom/totalfreedommod/discord/Discord.java

```diff
@@ -408,6 +411,15 @@ public void messageChatChannel(String message)
             message = StringUtils.remove(message, "§");
         }
 
+        // Patch FS-191 start
+        Matcher mention_matcher = this.discord_mention_pattern.matcher(message);
+
+        while (mention_matcher.find()) {
+            String mention = mention_matcher.group(1);
+            message = message.replace(mention, "[UserMention-redacted]");
```

Comment:
I think we should have something better than this.

Reply:
we could remove '@' from mention